### ISSUE TITLE
Missing  FORM Field LABELs - Main Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OSHA-Heat-Safety-iOS#1]
There is no INPUT field LABELs for the "Location", "Temperature", and "Humidity" fields to tell me what the information/different numbers mean for screen reader users. VoiceOver reads "edit field number", which doesn't tell me what the number should be.  

These LABELs can be visually hidden if that is part of the design.

WCAG 2.0 - 2.4.6 - Headings and Labels - Level AA

**Comment by MikePulsiferDOL** [comment]
Recommendations need to be UIKit-based, not HTML-based.

**Comment by skeeJay** [comment]
I can add "Temperature" and "Humidity" accessibility labels to these fields.

**Comment by jfc3-dol** [comment]
Apologies for using website based terms for hidden LABELs, when I should have been using the term accessibility LABELs.

VoiceOver is reading the accessibility LABELs correctly.

Closing Issue.
